app/validators/google_url_validator.rb

```diff
@@ -1,6 +1,6 @@
 class GoogleUrlValidator < ActiveModel::Validator
   def validate(record)
-    uri = URI.parse(record.url)
+    uri = URI.parse(record.url.strip)
```

Comment:
Perhaps we can pass in the stripped version into the model? Otherwise it'll still be saved in the db with the whitespace.